tools/run_tests/performance/scenario_config.py

```diff
@@ -31,6 +31,7 @@
 
 SINGLE_MACHINE_CORES=8
 WARMUP_SECONDS=5
+JAVA_WARMUP_SECONDS=15  # Java needs more warmup time for JIT to kick in.
```

Comment:
I agree that this is the best we can do for now, but Java really needs a 2-phase warmup. First at load for a period, then quiescent for a period (to give the JIT a chance to work). This needs driver support. I'll file an issue for this.